app-frontend/src/app/components/pages/project/layers/index.js

```diff
@@ -238,6 +238,39 @@ class ProjectLayersPageController {
         });
     }
 
+    zoomToSelected() {
```

Comment:
Since this is almost identical with what's in the analysis list, is there a possibility that it gets DRY'ed out in #4817?

Reply:
sounds like something worth doing. I'll add it to the list